packages/wonder-blocks-birthday-picker/src/components/birthday-picker.js

```diff
@@ -87,6 +97,11 @@ export const defaultLabels: Labels = Object.freeze({
     day: "Day",
 });
 
+// Default minWidth value when we include the full DOB.
+const FIELD_MIN_WIDTH_FULL = 110;
+// Alternative minWidth value when we hide the day field.
+const FIELD_MIN_WIDTH_MONTH_YEAR = 167;
```

Comment:
How are these values determined? Should we document a design reference? They seem arbitrary when all our other sizes are usually multiples of our spacing constants.